NYPLAudiobookToolkit/Utils/MediaProcessor.swift

```diff
@@ -300,4 +285,24 @@ class MediaProcessor {
             }
         }
     }
+    
+    private static func seek(filehandle: FileHandle, offset: UInt64) throws {
+        filehandle.seek(toFileOffset: offset)
+        // This is for when seek becomes stable. Currently these file operations seem to be unstable even though iOS13 deprecated them
+        // if #available(iOS 13.0, *) {
+        //     try fh.seek(toOffset: moov.offset)
+        // } else {
+        //     fh.seek(toFileOffset: moov.offset)
+        // }
+    }
+    
+    private static func offset(filehandle: FileHandle) throws -> UInt64 {
+        return filehandle.offsetInFile
```

Comment:
I was looking at the docs for these older apis and they are not marked as "throws" even though the docs said it will raise an exception... did you actually see an exception being raised? And does it translate to an actual Swift Error? Anyway, just curious.

Reply:
I don't know, but I assumed if it raised an error then I should wrap this with throws. Perhaps the implication by docs is that the error isnt throwable?